test/e2e/performanceprofile/functests/13_llc/llc.go

```diff
@@ -102,58 +105,109 @@ var _ = Describe("[Performance] Last Level Cache", Label(string(label.OpenShift)
 
 			By(fmt.Sprintf("Waiting when %s finishes updates", poolName))
 			profilesupdate.WaitForTuningUpdated(ctx, perfProfile)
+		}
+
+		// delete the machine config pool
+		err := testclient.Client.Delete(ctx, mc)
+		Expect(err).ToNot(HaveOccurred())
+
+		mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+		By("Waiting when mcp finishes updates")
+
+		mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+	})
+
+	Context("Configuration Tests", func() {
+
+		BeforeEach(func() {
+			if perfProfile.Annotations == nil || perfProfile.Annotations["kubeletconfig.experimental"] != llcPolicy {
+				testlog.Info("Enable align-cpus-by-uncorecache cpumanager policy")
```

Comment:
NIT: this is a CPU Manager policy option.

Reply:
fixed in the latest commit